libbeat/processors/config.go

```diff
@@ -125,8 +125,10 @@ func extractInt(unk interface{}) (uint64, error) {
 func extractString(unk interface{}) (string, error) {
 	switch s := unk.(type) {
 	case string:
-		return string(s), nil
+		return s, nil
+	case *string:
```

Comment:
I fixed the code that was producing the event containing the string pointer too. I added this code to just make it work in case someone makes this mistake in a community Beat.

> We shouldn't allow pointers in the condition of the processor.

How do we enforce this? Logging a warning isn't really strong enforcement. Ideally we could prevent a `*string` values from ever being written to the event, but this would require an actual event type with an API that's not a MapStr.